edb/server/mng_port/edgecon.pyx

```diff
@@ -1380,13 +1380,14 @@ cdef class EdgeConnection:
             if query_unit.system_config:
                 await self._execute_system_config(query_unit, conn)
             else:
-                await conn.parse_execute(
-                    query_unit,         # =query
-                    self,               # =edgecon
-                    bound_args_buf,     # =bind_data
-                    use_prep_stmt,      # =use_prep_stmt
-                    state,              # =state
-                )
+                if query_unit.sql:
+                    await conn.parse_execute(
```

Comment:
You probably also want to update the `simple_query()` method.

Reply:
OK, added `if query_unit.sql` guards everywhere where `edgecon` attempts to run it.